eZ/Publish/Core/Repository/URLService.php

```diff
@@ -245,9 +248,10 @@ protected function buildUpdateStruct(URL $url, URLUpdateStruct $data)
      * @param string $url
      *
      * @return bool
+     *
      * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
      */
-    protected function isUnique($id, $url)
+    protected function isUnique($id, $url): bool
```

Comment:
```suggestion
    protected function isUnique(int $id, string $url): bool
```

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/2854/commits/1719f1cf4a628cce40bbf098c2172c7ceae4d1f9